ckanext/ontario_theme/templates/internal/snippets/organization.html

```diff
@@ -1,11 +1,11 @@
 {% ckan_extends %}
 
 {% block heading %}
-  <h1 class="heading">{{ h.get_translated(organization, "title") or organization.name }}
+  <h3 class="module-heading">{{ h.get_translated(organization, "title") or organization.name }}
```

Comment:
I think we need a custom name for this heading. The designers would like this heading to be bigger than the Organization / Ministry `<h2>` tag, as it was before. [Figma comment](https://www.figma.com/file/LZ7QMXZVeKlkWn8qpV8FJ2?node-id=1399:1857#342666978)

The css styling used before : 
```margin: 0 0 5px 0;
    font-size: 18px;
    line-height: 1.3;
    -ms-word-break: break-all;
    word-break: break-all;
    word-break: break-word;
    -webkit-hyphens: auto;
    -moz-hyphens: auto;
    -ms-hyphens: auto;
    hyphens: auto;
```

Reply:
I made a class `ontario-module-h3` but some of the css was rejected, so I will omit them:

<img width="223" alt="Screen Shot 2023-01-11 at 3 17 40 PM" src="https://user-images.githubusercontent.com/1254764/211909009-5bb4ae9e-bf67-4255-a98e-d12845f52cbe.png">